frame/staking/src/tests.rs

```diff
@@ -2976,6 +2978,9 @@ fn claim_reward_at_the_last_era_and_no_double_claim_and_invalid_claim() {
 	// * an invalid era to claim doesn't update last_reward
 	// * double claim of one era fails
 	ExtBuilder::default().nominate(true).build_and_execute(|| {
+		// Consumed weight for all payout_stakers dispatches that fail
+		let err_weight = weights::SubstrateWeight::<Test>::payout_stakers_alive_staked(0);
```

Comment:
you should do the same patter in the main function as well, the code there is quite verbose.

Reply:
technically, it would be one more computation that is not needed if the 0 weight is not used...